model/testdata/dss/LICENSE

```diff
@@ -0,0 +1,502 @@
+                  GNU LESSER GENERAL PUBLIC LICENSE
```

Comment:
what file/s is/are under this license?

Reply:
Added a header

Reply:
@a5i It applies to all files under this folder testdata/dss ?